views/symptoms/rashes_meningitus.html

```diff
@@ -0,0 +1,316 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Rashes in babies and children" %}
+
+{% block localheader %}
+  <div id="local-header">
+    <div class="header-local-container">
+      <h1>{{ pagetitle }}</h1>
+      <div class="measure">
+        <p class="lede">Many things can cause a rash in children and they’re often nothing to worry about.</p>
+        <ul class="link-list">
+          <li><a href="#fever">Rash with fever</a></li>
+          <li><a href="#itching">Rash with itching</a></li>
+          <li><a href="#no-fever">Rash without fever or itching</a></li>
+        </ul>
+      </div>
+    </div>
+  </div>
+{% endblock %}
+
+{% block content %}
+
+  <div class="content">
+
+    <div class="grid-row">
+
+       <div class="grid-1-3 f-right">
+
+       <div class="callout aside emergency" id="callout-999">
+
+            <h3>Call 999 or go to A&amp;E if your child has a rash and they:</h3>
+
+            <ul>
+                <li>have a stiff neck</li>
+                <li>are bothered by light</li>
+                <li>seem confused</li>
+                <li>are shaking uncontrollably</li>
+                <li>have a fever you can’t control</li>
+                <li>has unusually cold hands and feet</li>
+                <li>have a rash that doesn’t fade when you press a <a href="#">glass against it</a></li>
+            </ul>
+
+            <p>These can be signs of <a href="http://www.nhs.uk/conditions/Meningitis/Pages/Introduction.aspx">meningitis</a>.</p>
+
+        </div>
+
+      </div>
+
+      <div class="grid-2-3">
+
+           <div class="callout gp">
+
+                <h3 class="heading-large">When to see a GP</h3>
+
+                <p>If your child seems unwell with a rash and has a fever call 111 or speak to your GP.</p>
+
+            </div>
+
+            <!--  Start main content -->
+            <p>The information below shouldn’t be used to self-diagnose your child’s condition, but it should give you a better idea of what is causing the rash. </p>
+
+            <h2 class="heading-xlarge" id="fever">Rash with fever</h2>
+
+            <div class="callout emergency" id="callout-999">
```

Comment:
There's 2 `div`'s with the id `callout-999`. This line and line 29. 

If it's there for an anchor link maybe just choose 1, or use different IDs.